src/main/java/seedu/address/model/event/EventTitle.java

```diff
@@ -0,0 +1,63 @@
+package seedu.address.model.event;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Event's title in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidEventTitle(String)}
+ */
+public class EventTitle implements Comparable<EventTitle> {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Event Title should only contain alphanumeric characters and spaces, and it should not be blank";
+
+    /*
+     * The first character of the address must not be a whitespace,
+     * otherwise " " (a blank string) becomes a valid input.
+     */
+    public static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
+
+    public final String eventTitle;
+
+    /**
+     * Constructs a {@code EventTitle}.
```

Comment:
```suggestion
     * Constructs an {@code EventTitle}.
```
Similar suggestion for grammar